### PR TITLE
Adding documentation for Locale

### PR DESCRIPTION
**Pull request by douglaseggleton**
Hey 👋🏻   Let me know what you think.

I went with the uppercasing of the country code in the end as thats the standard, we will accept any case though.
(Need to check in Embed).

### CODE REVIEW

File: guides/embed/locale.mdx [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+We currently support the following locales.
+
+| Locale Name          | Language Code (ISO 639-1) | Country Code (ISO 3166) | Locale |
+| -------------------- | ------------------------- | ----------------------- | ------ |
+| American English     | en                        | US                      | en-US  |
+| British English      | en                        | GB                      | en-GB  |
+| Brazilian Portuguese | pt                        | BR                      | pt-BR  |
+| French               | fr                        |                         | fr     |
+| Spanish              | es                        |                         | es     |
+| Portuguese           | pt                        |                         | pt     |
```
Comment (cbetta) [inline guides/embed/locale.mdx:67]: I'd put all these codes in backticks or the spellchecker will probably fail.

File: guides/embed/locale.mdx [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+Embed can be configured to use a different language using the `locale` property. By default, this will be set to `en` (English).
+Embed accepts a [ISO 639-1 Language Code](https://www.iso.org/iso-639-language-codes.html) (e.g. `en` or `fr`) and an optional [ISO 3166 Country Code](https://www.iso.org/iso-3166-country-codes.html) (e.g. `GB` or `NL`).
+The country code allows us to provide a localized version of the language based on a specific country, e.g. `fr-CA` would represent Canadian French.
+
+## Usage
+
+A locale from the list of [Supported Locales](#supported-locales) can be set.
+The locale property will fallback to English if a locale is not provided, is invalid or not supported.
+
+**Please Note: Embed currently does not try to detect the buyers locale automatically**.
```
Comment (cbetta) [inline guides/embed/locale.mdx:22]:
```suggestion
> Note: Embed currently does not automatically detect the buyer's locale. The
> locale will always be set to `en` when not provided.
```

File: guides/embed/locale.mdx [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+## Overview
+
+Embed can be configured to use a different language using the `locale` property. By default, this will be set to `en` (English).
+Embed accepts a [ISO 639-1 Language Code](https://www.iso.org/iso-639-language-codes.html) (e.g. `en` or `fr`) and an optional [ISO 3166 Country Code](https://www.iso.org/iso-3166-country-codes.html) (e.g. `GB` or `NL`).
+The country code allows us to provide a localized version of the language based on a specific country, e.g. `fr-CA` would represent Canadian French.
+
+## Usage
+
+A locale from the list of [Supported Locales](#supported-locales) can be set.
```
Comment (cbetta) [inline guides/embed/locale.mdx:19]:
```suggestion
The locale can be set to any of the [supported Locales](#supported-locales).
```

File: guides/embed/locale.mdx [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+The locale property will fallback to English if a locale is not provided, is invalid or not supported.
```
Comment (cbetta) [inline guides/embed/locale.mdx:20]:
```suggestion
The locale property will fall back to English if a locale is not provided, is incorrect or not supported.
```

File: guides/embed/locale.mdx [hunk 5]
```diff
@@ -0,0 +1,73 @@
...
+| Brazilian Portuguese | pt                        | BR                      | pt-BR  |
+| French               | fr                        |                         | fr     |
+| Spanish              | es                        |                         | es     |
+| Portuguese           | pt                        |                         | pt     |
+
+## FAQ
+
+### Will more locales be supported?
+
+We currently offer a limited set of locales and plan to roll-out more support in the future, including right-to-left languages such as Arabic.
```
Comment (cbetta) [inline guides/embed/locale.mdx:73]: I'd drop this until we fully make this decision.

**Review by cbetta — CHANGES_REQUESTED** [review]
Some small changes requested.

**Comment by douglaseggleton** [comment]
> Some small changes requested.

Done :-)

**Review by cbetta — APPROVED** [review]

**Pull request merged**